Gemfile

```diff
@@ -109,6 +109,8 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem "spring"
   gem "spring-watcher-listen", "~> 2.0.0"
+  gem 'spring-commands-rspec'
+  gem "pry-rails"
```

Comment:
small linting quality stuff....line 112 uses single quote and line 113 double quote. should be double quoted like everywhere here.